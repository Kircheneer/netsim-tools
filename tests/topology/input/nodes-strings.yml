defaults:
  device: iosv

nodes:
- e1
- e2
- e3
