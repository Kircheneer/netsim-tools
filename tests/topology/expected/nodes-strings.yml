input:
- topology/input/nodes-strings.yml
- package:topology-defaults.yml
name: input
nodes:
- box: cisco/iosv
  device: iosv
  id: 1
  loopback:
    ipv4: 10.0.0.1/32
  mgmt:
    ifname: GigabitEthernet0/0
    ipv4: 192.168.121.101
    mac: 08-4F-A9-00-00-01
  name: e1
- box: cisco/iosv
  device: iosv
  id: 2
  loopback:
    ipv4: 10.0.0.2/32
  mgmt:
    ifname: GigabitEthernet0/0
    ipv4: 192.168.121.102
    mac: 08-4F-A9-00-00-02
  name: e2
- box: cisco/iosv
  device: iosv
  id: 3
  loopback:
    ipv4: 10.0.0.3/32
  mgmt:
    ifname: GigabitEthernet0/0
    ipv4: 192.168.121.103
    mac: 08-4F-A9-00-00-03
  name: e3
provider: libvirt
